### ISSUE TITLE
train interrupt in the end of the first epoch

### ISSUE THREAD

**Issue by Surtr07** [mit-han-lab/bevfusion#381]
I train the model on 3 RTX3090 and finish the first epoch after about 10hours ,but its stuck on the end of the first epoch before save checkpoint. I use this command to train.
torchpack dist-run -np 3 python tools/train.py configs/nuscenes/det/transfusion/secfpn/camera+lidar/swint_v0p075/convfuser.yaml --model.encoders.camera.backbone.init_cfg.checkpoint pretrained/swint-nuimages-pretrained.pth --load_from pretrained/lidar-only-det.pth
<img width="761" alt="594954fa408f16cc301e1a284c16a0a" src="https://user-images.githubusercontent.com/109064691/231692278-b4e873c3-614d-4277-8050-0c582bb5087f.png">
<img width="334" alt="acdad41c9c3d50fffc722c05b0c53ce" src="https://user-images.githubusercontent.com/109064691/231692292-d82ba54e-5147-41b8-9c97-ed8418b91cc4.png">

**Comment by kentang-mit** [comment]
Hi @Surtr07,

Sorry for the delayed response. I was busy working on other projects recently. Judging from the snapshots you provided, it does not seem to me that you ran into a potentially OOM problem, and it seems that the GPUs still have high occupancies. You may add a `-v` flag to your command and see if there are any error messages shown on the screen when the program freezes. Besides, you could also try skipping the evaluation on validation set by setting [this parameter](https://github.com/mit-han-lab/bevfusion/blob/db75150717a9462cb60241e36ba28d65f6908607/configs/nuscenes/default.yaml#L298) to a very large value. Let me know if you have further information.

Best,
Haotian
